modules/installation-creating-gcp-vpc.adoc

```diff
@@ -40,19 +69,58 @@ resources:
 - name: cluster-vpc
   type: 01_vpc.py
   properties:
+ifndef::shared-vpc[]
     infra_id: '${INFRA_ID}' <1>
+endif::shared-vpc[]
+ifdef::shared-vpc[]
+    infra_id: '<prefix> '<1>
+endif::shared-vpc[]
     region: '${REGION}' <2>
     master_subnet_cidr: '${MASTER_SUBNET_CIDR}' <3>
     worker_subnet_cidr: '${WORKER_SUBNET_CIDR}' <4>
 EOF
 ----
+ifndef::shared-vpc[]
 <1> `infra_id` is the `INFRA_ID` infrastructure name from the extraction step.
+endif::shared-vpc[]
+ifdef::shared-vpc[]
+<1> `infra_id` is the prefix of the network name.
+endif::shared-vpc[]
 <2> `region` is the region to deploy the cluster into, for example `us-east1`.
 <3> `master_subnet_cidr` is the CIDR for the master subnet, for example `10.0.0.0/19`.
 <4> `worker_subnet_cidr` is the CIDR for the worker subnet, for example `10.0.32.0/19`.
 
 . Create the deployment by using the `gcloud` CLI:
 +
+ifndef::shared-vpc[]
 ----
 $ gcloud deployment-manager deployments create ${INFRA_ID}-vpc --config 01_vpc.yaml
 ----
+endif::shared-vpc[]
+ifdef::shared-vpc[]
+----
+$ gcloud deployment-manager deployments create ${INFRA_ID}-vpc --config 01_vpc.yaml --project ${HOST_PROJECT} --account ${HOST_PROJECT_ACCOUNT}
```

Comment:
Sorry for misleading you. As the variables INFRA_ID is not available here, we can change ${INFRA_ID}-vpc to <vpc_deployment_name> or something else. And we need to define HOST_PROJECT and HOST_PROJECT_ACCOUNT before this command line.